infra/nginx/nginx.conf

```diff
@@ -0,0 +1,35 @@
+user nginx;
+worker_processes  1;
+error_log stderr warn;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+    sendfile on;
+    keepalive_timeout 60;
+    gzip on;
+    client_max_body_size 100M;
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+    access_log stdout main;
+    proxy_cache_path /cache levels=1:2 keys_zone=my_cache:10m max_size=10g
+                     inactive=60m use_temp_path=off;
+    server {
+        listen *:80;
+        listen [::]:80;
+        server_name _;
+        location / {
+            proxy_pass http://vvgo:8080;
+            proxy_redirect ~https://$proxy_host:\d+(.*) https://$host$1;
+            proxy_set_header      Host            $host;
+            proxy_set_header      X-Real-IP       $remote_addr;
+            proxy_set_header      X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_connect_timeout 10;
+        }
+    }
```

Comment:
What's our TLS strategy? Terminate at ingress?

Reply:
Definitely terminate at ingress. I'm thinking we use cloudflare's certificates and have cloudflare tls to nginx. The only thing is that we need some way to share private keys. UGG that sounds annoying.

Reply:
SSL terminated cloud load balancer is the move for this.